docs/TASKS.md

```diff
@@ -3,6 +3,27 @@ Tasks
 
 A task describes the how, what, and when to do for a Snap job.  A task is described in a task _manifest_, which can be either JSON or YAML<sup>1</sup>.
 
+A task can be in the following states:
+- **started:** a running task
+- **stopped:** a task that is not running
+- **disabled:** a task in a state not allowed to start
```

Comment:
Is this not allowed to start? Or is it "stopped by the Snap"? This question is more for @intelsdi-x/snap-maintainers